extensions/eda/plugins/event_source/pg_listener.py

```diff
@@ -113,8 +113,10 @@ async def main(queue: asyncio.Queue[Any], args: dict[str, Any]) -> None:
                     await queue.put(data)
     except json.decoder.JSONDecodeError:
         LOGGER.exception("Error decoding data, ignoring it")
+        raise
```

Comment:
The previous line says: `Error decoding data, ignoring it` but we are not going to ignore it...

Reply:
fixed it